packages/block-editor/src/components/link-control/README.md

```diff
@@ -105,6 +105,13 @@ Whether to present suggestions when typing the URL.
 
 Whether to present initial suggestions immediately.
 
+### noDirectEntry
```

Comment:
Where is this used?

Reply:
Ah, that's unrelated to the other changes in that PR. It's a prop that was undocumented. I thought all props had to be in the docs, but happy to remove it from here if not.